site/static/examples/multikueue/external-frameworks-config.yaml

```
@@ -0,0 +1,19 @@
+metadata:
+  name: kueue-manager-config
+  namespace: kueue-system
+data:
+  multikueue:
+    gcInterval: 1m
+    origin: "multikueue-manager"
+    workerLostTimeout: 15m
+    externalFrameworks:
+      # Example 1: Tekton PipelineRun
+      - name: "PipelineRun.v1.tekton.dev"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)